server/statistics/hot_peer.go

```diff
@@ -127,15 +132,18 @@ func (stat *HotPeerStat) Log(str string, level func(msg string, fields ...zap.Fi
 		zap.Uint64("interval", stat.interval),
 		zap.Uint64("region-id", stat.RegionID),
 		zap.Uint64("store", stat.StoreID),
+		zap.Bool("is-leader", stat.isLeader),
```

Comment:
Do we really need this log with so many fields?

Reply:
I think we can remove some after #4461